Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.view.html

```diff
@@ -15,6 +15,33 @@ <h4 data-element-type="dpaMainDetailHeader">{{vm.headerName}}</h4>
                            autofocus
                            maxlength="150">
                 </div>
+                <!-- Agreement concluded -->
+                <div class="form-group">
+                    <label for="agreementConcluded" class="control-label small">Databehandleraftale indgået</label>
+                    <select class="form-control input-sm" id="agreementConcluded"
+                            ui-select2="noSearchNoClearSelect2"
+                            ng-model="vm.isAgreementConcluded.selectedElement"
+                            ng-change="vm.isAgreementConcluded.elementSelected(vm.isAgreementConcluded.selectedElement)"
+                            data-field="isAgreementConcluded"
+                            ng-disabled="!vm.hasWriteAccess">
+                        <option hidden selected value />
+                        <option ng-repeat="option in vm.isAgreementConcluded.select2Options" value="{{option.id}}" ng-bind-html="option.text" />
+                    </select>
+                </div>
+                <div ng-if="vm.isAgreementConcluded.selectedElement == 1">
```

Comment:
kan du ikke smide en enum ind i stedet for eller lave en viewmodel (din T) hvor der står "allowEditDate". så skal view bare fokusere på den og ikke på enum/number værdi.